docker/Dockerfile

```diff
@@ -20,7 +20,7 @@ ENV PYVER=${PYVER} \
 RUN ln -s /opt/python/cp${PYV}* /opt/python/v
 
 # install Python bindings and patch it to use the clang we have here
-RUN pip install future setuptools wheel clang==14.0 flake8 astunparse gast dm-tree black && \
+RUN pip install future setuptools wheel clang==14.0 flake8 astunparse gast dm-tree black==23.11.0 && \
```

Comment:
Adding this here stipulates, that this particular line shall be changed with every deps update, is that correct?

Reply:
yes

Reply:
Hopefully we'll not forget about this ;)

Reply:
That's why I put it in the third_party readme.